### PR TITLE
feat(ewd): Port EWD to MSFS Avionics framework

### PR DESCRIPTION
**Pull request by Eearslya**
## Summary of Changes

Ported the EWD from React over to MSFS avionics. This framework uses a (mostly) fixed DOM, allowing for better performance.

## Screenshots (if necessary)

Left: Original React EWD
Right: New MSFS Avionics EWD
image

## References

All references were taken from the existing EWD code.

## Additional context

Discord username (if different from GitHub): Eearslya#7831

## Testing instructions

Given that the EWD is a critical instrument with a lot of information, there are many things to be tested, and no single way of how to test them.
Things to test include:

- [ ] Engine status readouts (N1, EGT, N2, FF)
- [ ] Packs/Anti-Ice readouts
- [ ] N1 Limiter readout
- [ ] FOB readout
- [ ] Slats/Flaps indicator
- [ ] FWC Messages

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by auroraisluna** [comment]
do a ghost commit for title action to be resolved,  can I just add LOOKING FANCY. Have you measured the perf diff? Might be interesting to know

**Comment by Eearslya** [comment]
I've not done any measuring, no, but this seems to be the framework the team wants to move all of the instruments to!

**Review by Benjozork — CHANGES_REQUESTED** [review]
Not a complete review yet, but thought I'd leave some early comments

File: src/instruments/src/EWD2/EWD.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { LowerLeftDisplay } from './LowerLeftDisplay';
+import { LowerRightDisplay } from './LowerRightDisplay';
+
+import './style.scss';
+
+interface EWDProps extends ComponentProps {
+    bus: EventBus;
+    instrument: BaseInstrument;
+}
+export class EWDComponent extends DisplayComponent<EWDProps> {
```
[the remark this answers is not in the document]
Comment (Eearslya) [reply src/instruments/src/EWD2/EWD.tsx:15]: Should PFD be updated to follow this too?

[the remark this answers is not in the document]
Comment (Benjozork) [reply src/instruments/src/EWD2/EWD.tsx:15]: In a separate PR, if you want.

**Review by Benjozork — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7698
Tier of Testing  : 2
Date             : 13/02/2023

Testing Process: 
Things to test include:

 Engine status readouts (N1, EGT, N2, FF) - **OK**
 Packs/Anti-Ice readouts - **OK**
 N1 Limiter readout - **OK**
 FOB readout - **OK**
 Slats/Flaps indicator - **OK**
 FWC Messages - **OK**, didn't test all because that would be too much

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image

**Pull request merged**